dark-mode/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Let's not duplicate the license file

Reply:
@kb73 would you be able to update your branch with this change?